Dockerfile

```diff
@@ -21,5 +21,5 @@ RUN curl -sSL https://github.com/tcltk/tcllib/archive/release.tar.gz | tar -xz -
 
 RUN mkdir /logs
 WORKDIR /root
-RUN echo "$(crontab -l 2>&1; echo "*/5     *       *       *       *       /root/dns.tcl")" | crontab -
+RUN echo "$(crontab -l 2>&1; echo "*/60     *       *       *       *       /root/dns.tcl")" | crontab -
```

Comment:
Wont this mean that, if a users dynamic IP were to change, it could take up to an hour (probably two or three if you include time for DNS caching) for the DNS to be updated and services running from this DDNS IP to become usable again?

Reply:
That's correct. That's always been fine for me, but almost certainly isn't for everyone. I'd actually like this to be configurable so folks can tune it to their needs.